docs/en/compute/Hive.md

```diff
@@ -124,7 +124,7 @@ And you can see the query results from console:
 
 When Hive is already serving and managing the tables stored in HDFS,
 Alluxio can also serve them for Hive if HDFS is mounted as the under storage of Alluxio.
-In this example, we assume a HDFS cluster is mounted as the under storage of
+In this example, we assume an HDFS cluster is mounted as the under storage of
```

Comment:
okay i declare you to be the new grammar nazi. this is totally correct and i'm pretty sure it is incorrect in 90% of the cases... 👍